### ISSUE TITLE
Output directory structure

### ISSUE THREAD

**Issue by blitzrk** [blitzrk/sublime_libsass#8]
Should the default output structure be based on input structure or flat? Looking for comments.

I'm proposing a new config option to decide this. It would may have to handle two types of questions:
1. Flat or nested?
2. Where is the root of the nested folder relative to the `.libsass.json`?

Changes:
- New top-level option `output_style`
- `"flat"`, `"nested"`, or `["nested", "path/to/sass/root"]`
- Defaults to `"flat"`
- `"nested"` == `["nested", ""]`

e.g.

```
/proj/example/
|-- styles/
|   +-- scss/
|       |-- .libsass.json
|       |-- nested/
|       |   +-- module.scss
|       +-- main.scss
+-- static/
    +-- css/
        +-- main.css
```

``` json
{
    "output_dir": "../../static/css",
    "output_style": "nested",
    "options": {
        "style": "compressed"
    }
}
```

is equivalent to

```
/proj/example/
|-- .libsass.json
|-- styles/
|   +-- scss/
|       |-- nested/
|       |   +-- module.scss
|       +-- main.scss
+-- static/
    +-- css/
        +-- main.css
```

``` json
{
    "output_dir": "static/css",
    "output_style": ["nested", "styles/scss"],
    "options": {
        "style": "compressed"
    }
}
```

Output:

Either nested option creates:

```
/proj/example/
+-- static/
    +-- css/
        |-- nested/
        |   +-- module.css
        +-- main.css
```

Whereas flat creates:

```
/proj/example/
+-- static/
    +-- css/
        |-- main.css
        +-- module.css
```

@jbaf

**Comment by blitzrk** [comment]
I am somewhat worried that the name `output_style` will confuse users, because it overlaps with the `sassc` option for how the CSS will be formatted. In a similar vein, maybe `options` is too general, too.

Maybe config should look like

``` json
{
    "output": {
        "dir": "build/css",
        "structure": "flat"
    },
    "compile": {
        "style": "compressed",
        "line-comments": true
    }
}
```

**Comment by jbaf** [comment]
It looks like a great (and necessary) improvement for me.
And the new config file structure, if not mandatory, seems more accurate.

I look forward to these improvements.

**Comment by blitzrk** [comment]
Implemented in 0.9.0. Still need to update README.
